plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DefaultJdbcMetadataFactory.java

```diff
@@ -45,7 +45,7 @@ public JdbcMetadata create(JdbcTransactionHandle transaction)
                         jdbcClient,
                         Set.of(),
                         new SingletonJdbcIdentityCacheMapping(),
-                        new Duration(1, DAYS), true),
+                        new Duration(1, DAYS), true, 10000),
```

Comment:
Integer.MAX_VALUE here.

Reply:
@findepi Why? Wouldn't this mean that the configuration property is never used?
Or do you mean to say to change the default value to be Integer.MAX_VALUE? 

I want to confirm since I've asked @posulliv to change this to use the default value here (instead of MAX_VALUE).

Reply:
This is per-transaction cache, and transaction = query in JDBC connectors. Hence, for example, 1d hard-coded eviction time, which is "infinity in practice".
Is there any problem with not capping per-transaction cache with `cacheMaximumSize`?

The `cacheMaximumSize` will still be applied in `@Inject CachingJdbcClient::new`, which is used to configure permanent `CachingJdbcClient` (singleton).

Reply:
@findepi Thanks for pointing out this is per-transaction which I missed.
I think it should be fine to make per-transaction cache unbounded. It would help queries to see consistent metadata during runtime.

@posulliv Can you revert my suggestion and use `Integer.MAX_VALUE` here.